### PR TITLE
[The best] Timestamps are timestamps.

### PR DESCRIPTION
**Pull request by leogrim**
@ryanpbrewster @andrewconner

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/slack/models/SlackMessage.scala [hunk 1]
```diff
@@ -24,6 +25,7 @@ object SlackUsername {
 
 @json case class SlackMessageTimestamp(value: String) extends Ordered[SlackMessageTimestamp] { // channel-specific timestamp
   def compare(that: SlackMessageTimestamp) = value compare that.value
+  def toDatetime: DateTime = new DateTime(value.split(".").head.toLong * 1000) // "The bit before the . is a unix timestamp, the bit after is a sequence to guarantee uniqueness."
```
Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/slack/models/SlackMessage.scala]: `toDateTime`

**Comment by andrewconner** [comment]
Wahoo :+1:

File: kifi-backend/modules/shoebox/app/com/keepit/slack/SlackIngestionCommander.scala [hunk 2]
```diff
@@ -133,7 +133,7 @@ class SlackIngestionCommanderImpl @Inject() (
             isPrivate = None,
             canonical = None,
             openGraph = None,
-            keptAt = Some(clock.now()), // todo(Léo): not the best, should be the message timestamp, we do not get it back from Slack
+            keptAt = Some(message.timestamp.toDatetime), // todo(Léo): not the best, should be the message timestamp, we do not get it back from Slack
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/SlackIngestionCommander.scala]: comment no longer matters

File: kifi-backend/modules/shoebox/app/com/keepit/slack/SlackIngestionCommander.scala [hunk 3]
```diff
@@ -133,7 +133,7 @@ class SlackIngestionCommanderImpl @Inject() (
             isPrivate = None,
             canonical = None,
             openGraph = None,
-            keptAt = Some(clock.now()), // todo(Léo): not the best, should be the message timestamp, we do not get it back from Slack
+            keptAt = Some(message.timestamp.toDateTime), // todo(Léo): not the best, should be the message timestamp, we do not get it back from Slack
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/SlackIngestionCommander.scala]: comment no longer relevant

**Pull request merged**